refactor beats yaml config

zhongxp1219 opened the pull request.

zhongxp1219 committed:
bump pulsar-client version from 0.8.0 to 0.8.1

zhongxp1219 committed:
Merge branch 'streamnative:master' into master

zhongxp1219 committed:
1.create new sample config directory [sample/config/] to save beats yaml files
2.move filebeat.yml to sample/config/
3.update README.md

hezhangjian commented on README.md:
```diff
@@ -28,7 +28,9 @@ go build .
-#### Add following configuration to filebeat.yml
+In this section, you can use the sample config file in the directory [./sample/config/], or you can create it as follow steps.
+
+#### Add following configuration to filebeat.yml 
```
I think we can replace `filebeat.yml` to beat.yml. Since we already have metricbeat, heartbeat samples

jiazhai replied:
+1, filebeat.yml seems confusing. :)

Shoothzj commented:
Since we already release other beats packages. I think we can remove `Build other beat` from usage.

hezhangjian reviewed (changes requested):
Since we already release other beats packages. I think we can remove `Build other beat` from usage.

zhongxp1219 commented:
> Since we already release other beats packages. I think we can remove `Build other beat` from usage.

@Shoothzj  I think if we can reserve this section for others who want to `Build other beat` using source code ? Or we can remove usage section from README and let others follow our sample config - -

hezhangjian commented:
@zhongxp1219 It should be other section, not the usage section. Like `build` section

jiazhai commented:
Thanks, @zhongxp1219 for the great work. and thanks @Shoothzj for the review. 👍

zhongxp1219 committed:
1.create new sample config directory [sample/config/] to save beats yaml files
2.move filebeat.yml to sample/config/
3.update README.md

modified according to comments

hezhangjian commented:
I think the original `filebeat.yml` can be deleted @zhongxp1219

hezhangjian commented:
Great work.
